src/vue/common/IdleMessage.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div class="idle-message-wrapper">
+    <div class="idle-message">
+      <p class="idle-message__text">
+        {{ "idle-message.notification-message" | globalize }}
+      </p>
+      <div class="idle-message__btn-wrp">
+        <button class="idle-message__btn" @click="closeSelf">
+          {{ "idle-message.close-lbl" | globalize }}
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+const EVENTS = {
+  updateIsShown: 'update:isShown',
```

Comment:
component has no isShown property, but by emitting this one you assign it. remove this event